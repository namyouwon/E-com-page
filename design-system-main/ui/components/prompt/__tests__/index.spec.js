// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
/* eslint-env jest */

import createHelpers from '../../../../jest.helpers';
import Prompt from '../base/example';

const { matchesMarkup } = createHelpers(__dirname);

describe('Prompt', () => {
  it('renders a default prompt', () => matchesMarkup(Prompt));
});
