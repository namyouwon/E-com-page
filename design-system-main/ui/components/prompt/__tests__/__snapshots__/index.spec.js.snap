// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prompt renders a default prompt 1`] = `
<div
  className="demo-only"
  style={
    Object {
      "height": "24rem",
    }
  }
>
  <section
    aria-describedby="prompt-message-wrapper"
    aria-labelledby="prompt-heading-id"
    aria-modal="true"
    className="slds-modal slds-fade-in-open slds-modal_prompt"
    role="alertdialog"
    tabIndex="0"
  >
    <div
      className="slds-modal__container"
    >
      <div
        className="slds-modal__header slds-theme_error slds-theme_alert-texture"
      >
        <h1
          className="slds-text-heading_medium"
          id="prompt-heading-id"
        >
          Service unavailable
        </h1>
      </div>
      <div
        className="slds-modal__content slds-p-around_medium"
        id="prompt-message-wrapper"
      >
        <p>
          Sit nulla est ex deserunt exercitation anim occaecat. Nostrud ullamco deserunt aute id consequat veniam incididunt duis in sint irure nisi. Mollit officia cillum Lorem ullamco minim nostrud elit officia tempor esse quis. Cillum sunt ad dolore quis aute consequat ipsum magna exercitation reprehenderit magna. Tempor cupidatat consequat elit dolor adipisicing.
        </p>
      </div>
      <div
        className="slds-modal__footer slds-theme_default"
      >
        <button
          className="slds-button slds-button_neutral"
        >
          Got It
        </button>
      </div>
    </div>
  </section>
  <div
    className="slds-backdrop slds-backdrop_open"
    role="presentation"
  />
</div>
`;
